hkmc2/shared/src/main/scala/hkmc2/codegen/Block.scala

```diff
@@ -358,7 +355,7 @@ case class End(msg: Str = "") extends BlockTail with ProductWithTail
 
 enum Case:
   case Lit(lit: Literal)
-  case Cls(cls: ClassSymbol | ModuleSymbol, path: Path)
+  case Cls(cls: ClassLikeSymbol | ModuleSymbol, path: Path)
```

Comment:
Shouldn't this be
```suggestion
  case Cls(cls: ClassLikeSymbol, path: Path)
```
?